app/src/main/java/tech/relaycorp/gateway/domain/courier/DeliverParcel.kt

```diff
@@ -0,0 +1,17 @@
+package tech.relaycorp.gateway.domain.courier
+
+import tech.relaycorp.gateway.common.Operation
+import tech.relaycorp.gateway.data.model.MessageAddress
+import java.io.InputStream
+import javax.inject.Inject
+
+class DeliverParcel
```

Comment:
This will do the parcel delivery to a local endpoint, right? In that case, I think it should live outside `tech.relaycorp.gateway.domain.courier` because we'd need this code once we're able to sync over the Internet directly.

Reply:
Right, well spotted.